var/spack/repos/builtin/packages/netcdf-fortran/package.py

```diff
@@ -0,0 +1,171 @@
+from spack import *
+
+class NetcdfFortran(Package):
+    """Fortran interface for NetCDF4"""
+
+    homepage = "http://www.unidata.ucar.edu/downloads/netcdf/netcdf-cxx/index.jsp"
+    url      = "http://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-fortran-4.4.3.tar.gz"
+
+    version('4.4.3', 'bfd4ae23a34635b273d3eb0d91cbde9e')
+
+    variant('mpi', default=True, description='Enables MPI parallelism')
+
+    depends_on('netcdf')
+
+    def install(self, spec, prefix):
+        configure("--prefix=%s" % prefix)
+        make()
+        make("install")
+
+
+
+
+# netcdf-fortran configure parameters are below
+# ---------------------------------------------
+#
+# `configure' configures netCDF-Fortran 4.4.3 to adapt to many kinds of systems.
+# 
+# Usage: ./configure [OPTION]... [VAR=VALUE]...
+# 
+# To assign environment variables (e.g., CC, CFLAGS...), specify them as
+# VAR=VALUE.  See below for descriptions of some of the useful variables.
+# 
+# Defaults for the options are specified in brackets.
+# 
```

Comment:
These comments probably shouldn't be included since they will likely change, are easily obtainable, and don't appear in any other package in Spack.